### ISSUE TITLE
Verify current status of motor testing suite

### ISSUE THREAD

**Issue by Nicogene** [icub-tech-iit/ipts#17]
### Task description

During PI15 planning it came out that the motor testing suite is not complete.
We need to meet with @icub-tech-iit/proto in order to see which are the missing tests and plan the required activities.

### Definition of Done

Verification done + plan done.

**Comment by davidetome** [comment]
I got in touch w/ @Miche19 today and the problem spotted seems to be only the 2 missing BLDC motors codes missing (7116,7117) as anticipated yesterday.

In any case, the test support for those codes was ready from this release : 

- https://github.com/icub-tech-iit/ipts/releases/tag/v1.1.0

The problem was that they were using an old installer version of `IPTS`. 
⚠️ I strongly suggest using the latest releases of `IPTS`

Anyway, @Miche19 experienced some problems installing the latest release on its `Win10` machine due to a compatibility problem w/ the last `.NET` framework used.

I fixed downgrading the `.NET` target framework to `4.5.1` : 

-  https://github.com/icub-tech-iit/ipts/pull/18

So I put this in review and kindly ask to @Fabrizio69 if there is any other missing stuff.

cc @Nicogene @pattacini @GiacomoVassallo13
